kylinpy/sqla_dialect.py

```diff
@@ -42,11 +42,6 @@ class KylinSQLCompiler(compiler.SQLCompiler):
     def __init__(self, *args, **kwargs):
         super(KylinSQLCompiler, self).__init__(*args, **kwargs)
 
-    def _compose_select_body(self, text, select, inner_columns, froms, byfrom, kwargs):
-        text = super(KylinSQLCompiler, self)._compose_select_body(
-            text, select, inner_columns, froms, byfrom, kwargs)
-        return text
-
```

Comment:
the SQLAlchemy latest version has changed [_compose_select_body](https://github.com/sqlalchemy/sqlalchemy/blob/b3bb55aaea9ee94e50f8db594d65c2bfcdb314ef/lib/sqlalchemy/sql/compiler.py#L3652) signatures so the sub-class wasn't able to adapt the superclass. 

`_compose_select_body` has no extra functionality, so it can be safely removed in subclasses.